### PR TITLE
Fetch global contributor list from redis

### LINKED ISSUE #240: Dynamic contributors list on the global about page
When people help to clean missions we add them to the mission's `_meta` transcript (Kallisto builds this list for us when the mission's been cleaned there). We also add people to the global site's about page, but we've been somewhat inconsistent about doing that.

Instead of updating the global site by hand every time, let's just pull in the list from Redis.

### PR DESCRIPTION
**Pull request by jaylett**
* pull from mission cleaners list
 * have to remove original team explicitly
 * we also carry a list of code-only contributors
 * cleaners don't have URLs but contributors can, so that's more
   data in code

Closes #240.

### CODE REVIEW

File: global/apps/homepage/views.py [hunk 1]
```diff
@@ -146,11 +146,77 @@ def _get_reading_list(country_code):
 
 @cache_control(no_cache=True)
 def about(request):
+    country_code = request.META.get('HTTP_CF_IPCOUNTRY')
+    if country_code is None:
+        country_code = request.META.get(
+            'GEOIP_COUNTRY_CODE',
+            '--',
+        )
```
Comment (jaylett) [inline global/apps/homepage/views.py]: I've just realised that this has nothing to do with the globa contributor list. I should separate it at least into different commits, if not different PRs.

File: global/apps/homepage/views.py [hunk 2]
```diff
@@ -146,11 +146,77 @@ def _get_reading_list(country_code):
...
+        'Chris Govias',
+        'Gavin O\'Carroll',
+        'Ryan Alexander',
+        'James Aylett',
+        'George Brocklehurst',
+        'David Brownlee',
+        'Ben Firshman',
+        'Mark Norman Francis',
+        'Andrew Godwin',
+        'Steve Marshall',
```
Comment (georgebrock) [inline global/apps/homepage/views.py]: We've now got the original team listed in the view and in the template. Is it worth passing our names to the template too? Maybe not, since it's never going to change.

File: global/apps/homepage/views.py [hunk 3]
```diff
@@ -146,11 +146,77 @@ def _get_reading_list(country_code):
...
+            'GEOIP_COUNTRY_CODE',
+            '--',
+        )
+
+    # Fetch the list of contributors from the various missions. Use a
+    # set() because we only want to list each contributor once.
+    contributors = set()
+    for mission in list(Mission.Query(request.redis_conn)):
+        if not mission.incomplete:
+            contributors = contributors | set(mission.copy.get('cleaners', []))
```
Comment (georgebrock) [inline global/apps/homepage/views.py]: Why do we need to copy the mission here? We don't seem to be mutating the mission, and even if we were, we don't seem to be using the missions again after this loop.

Comment (jaylett) [inline global/apps/homepage/views.py]: `mission.copy` is a member, not a `copy()` method.

File: global/apps/homepage/views.py (hunk 2 above)
Comment (jaylett) [inline global/apps/homepage/views.py]: Yeah, I can't see it worth fixing that without also replacing the way we do things for the website, which is more work for something that won't change.

File: global/apps/homepage/views.py (hunk 3 above)
Comment (georgebrock) [inline global/apps/homepage/views.py]: Oops. Too much Ruby lately!
